docs/installation/enabling-control-service.rst

```diff
@@ -29,6 +29,27 @@ For more details on configuring the firewall, see the `FirewallD documentation`_
 
 On AWS, an external firewall is used instead, which will need to be configured similarly.
 
+RHEL 7.2
+========
+
+.. prompt:: bash root@rhel:~$
```

Comment:
The other instructions in this document use prompt directives using root - should this be `.. prompt:: bash [root@rhel]#?`